release.json

```diff
@@ -22,6 +22,30 @@
         "WINDOWS_DDNPM_SHASUM": "61e13fccdca7bf6a356e7ef552a7845316b9905b5eee264811982e3034a3bb84",
         "SECURITY_AGENT_POLICIES_VERSION": "master"
     },
+    "6.28.1-rc.1": {
+        "INTEGRATIONS_CORE_VERSION": "7.28.1",
+        "OMNIBUS_SOFTWARE_VERSION": "7.28.1-rc.1",
+        "OMNIBUS_RUBY_VERSION": "7.28.1-rc.1",
+        "JMXFETCH_VERSION": "0.44.0",
+        "JMXFETCH_HASH": "758717a2fd9c50247336318644e9c985d9a8a9845039d59a8793923b3f76f494",
+        "SECURITY_AGENT_POLICIES_VERSION": "v0.9",
+        "MACOS_BUILD_VERSION": "6.28.1-rc.1",
+        "WINDOWS_DDNPM_DRIVER": "release-signed",
+        "WINDOWS_DDNPM_VERSION": "0.98.2.git.68.90e7d1e",
+        "WINDOWS_DDNPM_SHASUM": "f6efeb772e011d7a5327b413cc60e5380c6d108ca61554803c8e6c3fadcf14c1"
+    },
+    "7.28.1-rc.1": {
+        "INTEGRATIONS_CORE_VERSION": "7.28.1",
+        "OMNIBUS_SOFTWARE_VERSION": "7.28.1-rc.1",
```

Comment:
were there changes to the omnibus-software and omnibus-ruby projects? If not, I don't think we need new tags there, it's prone to error.

Reply:
- `omnibus-ruby` has the same commit for all tags (7.28.0, 7.28.1, and 7.28.1-rc.1)
- `omnibus-software` has 1 commit diff between 7.28.0 and others but it seems like a desired change: https://github.com/DataDog/omnibus-software/pull/396/files